store/storetest/team_store.go

```diff
@@ -1416,14 +1416,13 @@ func testTeamStoreMigrateTeamMembers(t *testing.T, ss store.Store) {
 	lastDoneUserId := strings.Repeat("0", 26)
 
 	for {
-		res := <-ss.Team().MigrateTeamMembers(lastDoneTeamId, lastDoneUserId)
-		if assert.Nil(t, res.Err) {
-			if res.Data == nil {
+		res, _ := ss.Team().MigrateTeamMembers(lastDoneTeamId, lastDoneUserId)
+		if assert.Nil(t, res) {
```

Comment:
i think we need to store the `err` value from above and verify if `err` is `nil` here as opposed to checking if `res` is `nil` here.

Reply:
@ali-farooq0 thank you. Let me update the code.